src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -517,4 +517,13 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
             this._stripeElement = undefined;
         }
     }
+
+    private _getAddress(isBopisEnabled: boolean): Address | undefined | null {
```

Comment:
This doesn't return an address at all. Does this seem correct?

Reply:
Yes, I intentionally send `null`, please see my comment above.

Reply:
Sorry think I didn't explain clearly, the function here doesn't return `Address` in any path, it either returns undefined or null?